lib/swaps/Swaps.ts

```diff
@@ -256,7 +257,7 @@ class Swaps extends EventEmitter {
   public executeSanitySwap = async (currency: string, peer: Peer) => {
     const { rPreimage, rHash } = await generatePreimageAndHash();
     const peerPubKey = peer.nodePubKey!;
-    const swapClient = this.swapClients.get(currency);
+    const swapClient = this.swapClientManager.get(currency);
```

Comment:
We probably want a try/catch here (rather than the if statement below) in case we have no swap client for this currency.